applications/Identity_wallet_and_pallet.md

```diff
@@ -0,0 +1,302 @@
+
+The Substrate / Polkadot and Kusama ecosystems have shown significant interest in Decentralized Identity as there are several identity-related projects, pallets and chains such as the [DID Pallet](https://github.com/substrate-developer-hub/pallet-did), Identity Pallet,  [Parami](https://github.com/parami-foundation/parami-blockchain), [KILT](https://github.com/KILTprotocol/kilt-node) and [Dock](https://github.com/docknetwork/dock-substrate). While the projects are great, there is some major untapped potential. With Decentralized Identity, web3 users can easily authenticate themselves and create trust between each other. This can be immensely valuable to web3 (and web2) service providers, for user authentication and identification. As Impierce Technologies, we do currently warn of the potential of GDPR risks with some of these implementations as writing any Personal Identifiable Information to a Blockchain is most likely a irreversible GDPR violation. With this proposed project we provide a GDPR compliant alternative for the entire Substrate ecosystem.
+
+### Project Details
+
+Impierce Technologies would like to develop and maintain an Identity Wallet in several phases over the following years. This initial phase requires us to implement an Identity Wallet with all features necessary for users to create and manage their substrate-based DIDs, log-in and authenticate yourself with Relying Parties and collect and share Verifiable Credentials. In later phases, we would like to further enhance the wallet with more features that will connect ecosystems and encourage adoption.
+
+In order to achieve the first phase, we would develop a mature DID Pallet and accompanying Substrate DID Method that fully complies with the DID standard. In addition, we implement a client that can connect to an Identity Provider that is compatible with the SIOPv2, OIDC, OpenID4VP and OpenID4VCI. These are OpenID Foundation standards that allow user authentication following globally recognized standards adopted by hundreds of thousands of websites.
+
+The Identity Wallet is a sensitive piece of software as it will handle private keys and personal information. As such, security is our topmost priority. We have decided to develop the cross-platform app in Rust using the [Tauri framework](https://tauri.app/). It is chosen as it provides the best tools to our developers for separating front-end rendering from business logic and handling sensitive data. This clear separation allows us to handle any sensitive information inside Rust, which gives an extra layer of defense in order to maintain optimal security and privacy.
+
+![Overview of the Components](https://i.imgur.com/VcSkujW.png)

```

Comment:
It is indeed simplified. The single lines are actual bi-directional moments of communication that require multiple steps. I just wanted to visualize from a high-level the components and standards and what role they fulfill. It has become easier since your time at uPort. It used to be the wildwest in how an Identity Wallet would communicate with Services, but due to EU legislative developments, OpenID standards have gained a lot of favor and have also matured very nicely. So it is still a significant undertaking, but it is mostly implementing standards. This definitely also favors adoption as most SSI projects will start to shift towards these standards. 

A more detailed diagram would basically be a diagram of the different OpenId message standards, which I have all linked in the document. I can make a more detailed diagram if that helps, but it will purely be a summary of these standards, nothing we would design or architect.